tests/integration_tests/test_ingest_db_transactions.py

```diff
@@ -601,7 +610,7 @@ def test_next_submission_numpy_type(test_client_rollback):
     prog = Programme(
         programme_id="HS-ROW",
         programme_name="TEST-PROGRAMME-NAME1",
-        fund_type_id="HS",
+        fund_type_id=Fund.query.first().id,
```

Comment:
It'd probably be nice for readability to pin this to specifically be the HS fund PK? Probably as a static value if we can define one for it.